### ISSUE TITLE
Skiovox helper: manifest.json blocked by admin

### ISSUE THREAD

**Issue by FunnyDoggs** [bypassiwastaken/skiovox-helper#123]
Tried to reload the extension when it wasn't working, and now it says its blocked.

**Comment by FunnyDoggs** [comment]
![Screenshot 2024-04-10 8 25 33 AM](https://github.com/bypassiwastaken/skiovox-helper/assets/150701373/22007c56-b2ca-40b9-9217-9b82a4b51d51)

**Comment by FunnyDoggs** [comment]
following the instructions fixed it
https://www.plasmo.com/blog/posts/how-to-create-a-consistent-id-for-your-chrome-extension

**Comment by FunnyDoggs** [comment]
It didn't fix the new tab though

**Comment by FunnyDoggs** [comment]
fixed the version in the manifest, and background.scripts to background.service_workers, am now getting
![Screenshot 2024-04-10 8 40 47 AM](https://github.com/bypassiwastaken/skiovox-helper/assets/150701373/9b9443aa-89ba-4b4c-9f17-0762142850bc)

**Comment by FunnyDoggs** [comment]
https://stackoverflow.com/questions/66851301/permission-all-urls-is-unknown-or-url-pattern-is-malformed

dont know exactly where to put host_permissions

**Comment by FunnyDoggs** [comment]
![Screenshot 2024-04-10 8 46 12 AM](https://github.com/bypassiwastaken/skiovox-helper/assets/150701373/a9560a44-1eb5-46c9-98c2-39c206ca97c0)

**Comment by FunnyDoggs** [comment]
Full log so far:
Permission '<all_urls>' is unknown or URL pattern is malformed.
{
  "manifest_version": 3
,
  "version": "2.4.1",
  "name": "Skiovox Helper",
  "description": "A helper for the Skiovox exploit -- read more at skiovox.com",
  "icons": {
    "128": "icon.png"
  },
  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwnmXkLn2EnRoDsMdqeCx7Um81I6Bj+mjw/zOITS8eEKgmvsNB+tVpg3Z+ll/i0uqbTWSYqFiSwhUZWCaQB6eGZtenXa5w3Bk0+81tYG7i6S+osk2frIYCf33qxlR1EdDI+epyF4qTrrezWoNKSAHlQ1qbW0gwZGPN1zpGOWjiysymAJlCwOU4s3Q2tZQXkEGPm4g1RhU6PN7xjnUGz5v3lnRlOM4+wicUQGw4MG6F0gSPaRy4k8tHOiiaqPvuj0dy8uYUMsOIQAgUfm9qh04OyoXChfJp/j/3C52a1M6ixCTtxz1muCyaQCgzbhFdrP2WMr+KlCrl53NrmHZ8RRjMwIDAQAB",
  "commands": {
    "ACCESS_HISTORY": {
      "description": "Ctrl+H"
    },
    "ACCESS_DOWNLOADS": {
      "description": "Ctrl+J"
    },
    "NEW_TAB": {
      "description": "Ctrl+T"
    },
    "CLOSE_TAB": {
      "description": "Ctrl+W"
    },
    "RESTORE_TAB": {
      "description": "Ctrl+Shift+T"
    },
    "NEW_WINDOW": {
      "description": "Ctrl+N"
    },
    "NEW_INCOG_WINDOW": {
      "description": "Ctrl+Shift+N"
    },
    "CLOSE_WINDOW": {
      "description": "Ctrl+Shift+W"
    },
    "TAB_NEXT": {
      "description": "Cycle forwards through tabs: Ctrl+Q"
    },
    "TAB_BACK": {
      "description": "Cycle backwards through tabs: Ctrl+Shift+Q"
    },
    "SWITCH_WINDOWS": {
      "description": "Switch windows: Alt+Q"
    },
    "CTRL_1": {
      "description": "Ctrl+1"
    },
    "CTRL_2": {
      "description": "Ctrl+2"
    },
    "CTRL_3": {
      "description": "Ctrl+3"
    },
    "CTRL_4": {
      "description": "Ctrl+4"
    },
    "CTRL_5": {
      "description": "Ctrl+5"
    },
    "CTRL_6": {
      "description": "Ctrl+6"
    },
    "CTRL_7": {
      "description": "Ctrl+7"
    },
    "CTRL_8": {
      "description": "Ctrl+8"
    },
    "CTRL_9": {
      "description": "Ctrl+9"
    },
    "VIEW_SOURCE": {
      "description": "Ctrl+U"
    }
  },
  "background": {
    "service_workers": [
      "/background/commands.js",
      "/background/setup.js",
      "/background/webstore-fix.js"
    ]
  },
  "chrome_url_overrides": {
    "newtab": "/new-tab/main.html"
  },
  "permissions": [
    "commands",
    "tabs",
    "sessions",
    "<all_urls>"
  ],
  "content_scripts": [
    {
      "matches": [
        "*://*/*"
      ],
      "js": [
        "/content-scripts/window-open-fix.js"
      ],
      "all-frames": true,
      "run-at": "document_start"
    }
  ]
}

Refused to load the script 'blob:https://www.google.com/21edc4c7-7a82-4b88-a0dd-c6e5eda4a4d2' because it violates the following Content Security Policy directive: "script-src 'self' 'wasm-unsafe-eval' 'inline-speculation-rules' http://localhost:* http://127.0.0.1:*". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
Context
https://www.google.com/search?q=Permission+%27%3Call_urls%3E%27+is+unknown+or+URL+pattern+is+malformed.&rlz=1CATAVM_enUS1103&oq=Permission+%27%3Call_urls%3E%27+is+unknown+or+URL+pattern+is+malformed.&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIICAEQABgFGB7SAQc2MzZqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8
Stack Trace
content-scripts/window-open-fix.js:10 (anonymous function)

function fixWindowOpen() {
    let nativeOpen = window.open
    window.open = (a, b) => nativeOpen(a, b) // strip third param
}

let blob = new Blob([`(${fixWindowOpen})()`], { type: "text/javascript" })

let script = document.createElement('script')
script.src = URL.createObjectURL(blob)
document.body.appendChild(script)

Refused to load the script 'blob:https://stackoverflow.com/f640a515-6198-4539-be83-a71802064eca' because it violates the following Content Security Policy directive: "script-src 'self' 'wasm-unsafe-eval' 'inline-speculation-rules' http://localhost:* http://127.0.0.1:*". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
Context
https://stackoverflow.com/questions/66851301/permission-all-urls-is-unknown-or-url-pattern-is-malformed
Stack Trace
content-scripts/window-open-fix.js:10 (anonymous function)

function fixWindowOpen() {
    let nativeOpen = window.open
    window.open = (a, b) => nativeOpen(a, b) // strip third param
}

let blob = new Blob([`(${fixWindowOpen})()`], { type: "text/javascript" })

let script = document.createElement('script')
script.src = URL.createObjectURL(blob)
document.body.appendChild(script)

Refused to load the script 'blob:https://developer.chrome.com/4b5122bc-580f-44be-b674-3c57430689d8' because it violates the following Content Security Policy directive: "script-src 'self' 'wasm-unsafe-eval' 'inline-speculation-rules' http://localhost:* http://127.0.0.1:*". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
Context
https://developer.chrome.com/docs/extensions/develop/migrate
Stack Trace
content-scripts/window-open-fix.js:10 (anonymous function)
function fixWindowOpen() {
    let nativeOpen = window.open
    window.open = (a, b) => nativeOpen(a, b) // strip third param
}

let blob = new Blob([`(${fixWindowOpen})()`], { type: "text/javascript" })

let script = document.createElement('script')
script.src = URL.createObjectURL(blob)
document.body.appendChild(script)

**Comment by FunnyDoggs** [comment]
New tab works but shortcuts don't

**Comment by FunnyDoggs** [comment]
even after set

**Comment by fowntain** [comment]
Hey, this is because the extension uses manifest V2 for permissions reasons. As long as you update the public key it should work fine. My SD blocked the ID, so I updated the public key and didn't have any issues. The old version will start causing problems starting in June, when Google deprecates V2...

**Comment by FunnyDoggs** [comment]
Yeah I did the same. It works, but not with the shortcuts and I have to load the extension as a new one each time.

**Comment by FunnyDoggs** [comment]
With the load unpacked button.

**Comment by FunnyDoggs** [comment]
It wouldn't work without changing the manifest_version tag from 2 to 3.

**Comment by FunnyDoggs** [comment]
I don't know exactly why that is. I have CAUB set up, so Chrome can't have updated.

**Comment by FunnyDoggs** [comment]
I guess it is a verification that is checked against server data?

**Comment by bbg37** [comment]
Hey FunnyDoggs, the same thing happened to me, and I assume you got it to work? Could you walk me through the steps on how you did it? I can't seem to figure it out?

**Comment by VFXKLIPS** [comment]
> Tried to reload the extension when it wasn't working, and now it says its blocked.

Can you tell me how to fix it please respond fast

**Comment by VFXKLIPS** [comment]
How do i use skebstore

**Comment by VFXKLIPS** [comment]
> Tried to reload the extension when it wasn't working, and now it says its blocked.


When i try it it says

(Failed to load extension File ~/MyFiles/Downloads/shkrmivox-helper-main (1) Error Manifest file is missing or unreadable Could not load manifest.)

Can you pls tell me how to fix it pls

Or tell me how to use Skebstore
